applications/GeoMechanicsApplication/custom_strategies/schemes/backward_euler_scheme.hpp

```diff
@@ -52,13 +53,13 @@ class BackwardEulerScheme : public GeoMechanicsTimeIntegrationScheme<TSparseSpac
         KRATOS_TRY
 
         block_for_each(rModelPart.Nodes(), [this](Node& rNode) {
-            // For the Backward Euler schemes the first derivatives should be
```

Comment:
A similar comment is already placed in line 76, so I thought it was redundant to leave it here